Model/Source/DefaultOptions.php

```diff
@@ -102,8 +116,8 @@ public function getDefault($option)
      */
     public function getMaxCompanyName(?string $company): ?string
     {
-        if (null !== $company && (strlen($company) > self::COMPANY_NAME_MAX_LENGTH)) {
-            $company = substr($company, 0, self::COMPANY_NAME_MAX_LENGTH - 3) . '...';
+        if ($company !== null && (strlen($company) >= self::COMPANY_NAME_MAX_LENGTH)) {
+            $company = substr($company, 0, 47) . '...';
```

Comment:
Dit is niet goed in de NL versie (en dus nu ook niet hier), gebruik Str::limit met de COMPANY_NAME_MAX_LENGTH constante, die hoeft niet 50 te zijn!